onchain/rollups/contracts/consensus/quorum/IQuorumFactory.sol

```diff
@@ -0,0 +1,61 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+pragma solidity ^0.8.8;
+
+import {Quorum} from "./Quorum.sol";
+import {IHistory} from "../../history/IHistory.sol";
+
+/// @title Quorum Factory interface
+interface IQuorumFactory {
+    // Events
+
+    /// @notice A new quorum was deployed.
+    /// @param quorum The quorum
+    /// @param history The history
+    /// @dev MUST be triggered on a successful call to `newQuorum`.
+    event QuorumCreated(Quorum quorum, IHistory history);
```

Comment:
Why not emit `validators` and `shares` as well?

Reply:
Because they were already emitted in the Quorum contract via PaymentSplitter, right? 
ref: https://github.com/binodnp/openzeppelin-solidity/blob/ffc4bcbba398b61d19ede03e4fdf673ddbce1083/contracts/payment/PaymentSplitter.sol#L114C35-L114C35

Reply:
Okay, fair enough.